### ISSUE TITLE
sphere_ssp_baroclinic_wave_rhoe_equilmoist_conservation_ft64 is unstable

### ISSUE THREAD

**Issue by szy21** [CliMA/ClimaAtmos.jl#1406]
See this [build](https://buildkite.com/clima/climaatmos-ci/builds/7601#018694a8-08dd-4e0e-a0a2-625d21f65fc3) for example. Reducing the timestep from 450secs to 300secs doesn't help.

This experiment is not very physical. It is a moist baroclinic wave without precipitation (0M microphysics) for checking mass and energy conservation. As a result of no precipitation, q_tot can become much higher than the saturation specific humidity. I don't know why it works fine for ARS with no limiter, but doesn't work for SSP with limiter though.

I think we can move on with the SSP/limiter/upwinding testing without worrying about this test, as long as the statistically steady state of a moist simulation looks fine. @dennisYatunin @valeriabarra @tapios Any thoughts?

**Comment by valeriabarra** [comment]
I also don't have other explanations, other than the fact that the test is not very physical, as you said. 
I tried to run it with `--precip_model 0M` and it runs to completion (obviously in that case the conservation check fails, but only by very little O(1e-7)). 
I am not sure why this would not happen with the ARS time-stepper. Perhaps in that case the fact that `qt` was going slightly negative was helping in not making it increase too much, causing instabilities? 
Perhaps @dennisYatunin has more insights on the different behaviors between the two time-steppers.

**Comment by dennisYatunin** [comment]
Yeah other than decreasing dt for SSP333 I got nothing. Maybe using a limiter is throwing off the convergence, so we need to decrease dt even more?

**Comment by szy21** [comment]
This test has been removed from ci, so I'm closing this issue.
